doc/modules/ROOT/pages/interceptors/interceptors.adoc

```diff
@@ -53,6 +53,10 @@ exists.
 The `ConfigValueConfigSourceInterceptor` is not registered by default. You need to register it via the ServiceLoader
 mechanism with your application.
 
+WARNING: The `ConfigValueConfigSourceInterceptor` may log sensitive information present in your configuration, so be
```

Comment:
TBH if we're going to do this, we should properly address secrets in a global way instead?

Reply:
I believe this is the first time we have something that can output the config value directly.

Yes, maybe we should address that in a more global way, but I think it is out of scope of this PR. On the other hand, I didn't want to leave this completely "open".